### PR TITLE
Ani 2x

### PR DESCRIPTION
**Pull request by cdever01**
Added ANI-2x parameters

### CODE REVIEW

**Comment by farhadrgh** [comment]
Would you please update [`models.py`](torchani/models.py) to include `ANI2x` class

File: torchani/models.py [hunk 1]
```diff
@@ -282,3 +282,19 @@ def ANI1ccx(periodic_table_index=False, model_index=None):
     if model_index is None:
         return BuiltinEnsemble._from_neurochem_resources(info_file, periodic_table_index)
     return BuiltinModel._from_neurochem_resources(info_file, periodic_table_index, model_index)
+
```
Comment (zasdfgbnm) [inline torchani/models.py:285]: @IgnacioJPickering You recently changed this file, so please review

**Comment by zasdfgbnm** [comment]
After this is ready, we should merge and make a new release. 

https://github.com/aiqm/torchani/pull/471 should block on this, and can only be merged after the release. I don't want a new release with such an important thing like ANI2x to be compatible with only pytorch versions as new as 20200606.

**Comment by IgnacioJPickering** [comment]
Isn't ANI-2x not public yet?

**Comment by zasdfgbnm** [comment]
@IgnacioJPickering I don't know, but the paper is already on chemrxiv?

File: torchani/models.py (hunk 1 above)
Comment (IgnacioJPickering) [reply torchani/models.py:285]: The function LGTM, *but* ANI 1x and ANI 1ccx have 339 files each, why does this PR modify 557 files? @cdever01

Comment (farhadrgh) [reply torchani/models.py:285]: There are a few problems with doc strings in these new changes

Comment (IgnacioJPickering) [reply torchani/models.py:285]: Yes you are right. @cdever01 told me that the new files come from the new atom types so that should be fine

**Comment by IgnacioJPickering** [comment]
According to @cdever01 its fie to upload, and the number of files is correct.

**Comment by farhadrgh** [comment]
@zasdfgbnm, @IgnacioJPickering Docs are ready, You can find the link to ChemRxiv paper too

**Pull request merged**